tests/python/contrib/test_hexagon/conv_uint8.py

```diff
@@ -0,0 +1,170 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
```

Comment:
any reason we should keep these files(conv_uint8.py and conv_uint8_hvx...) here and not in the hvx directory?